From 207511fad218f3e053516864b2e2cf867f0eb9d2 Mon Sep 17 00:00:00 2001
From: Lukas Czerner <lczerner@redhat.com>
Date: Wed, 29 Oct 2014 16:34:30 -0400
Subject: [fs] ext4: explicitly inform user about orphan list cleanup

Message-id: <1414600485-1929-41-git-send-email-lczerner@redhat.com>
Patchwork-id: 98547
O-Subject: [RHEL 7.1 PATCH v2 40/55] ext4: explicitly inform user about orphan list cleanup
Bugzilla: 1156096
RH-Acked-by: Eric Sandeen <sandeen@redhat.com>

From: Dmitry Monakhov <dmonakhov@openvz.org>

BZ 1156096
https://bugzilla.redhat.com/show_bug.cgi?id=1156096

BREW 8169722
https://brewweb.devel.redhat.com/taskinfo?taskID=8169722

Upstream commit 844749764b416ee2c4ba2da328c04eaad7388242

 Production fs likely compiled/mounted w/o jbd debugging, so orphan
 list clearing will be silent.

 Signed-off-by: Dmitry Monakhov <dmonakhov@openvz.org>
 Signed-off-by: Theodore Ts'o <tytso@mit.edu>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/fs/ext4/super.c b/fs/ext4/super.c
index 6c08ff8..a0bfafe 100644
--- a/fs/ext4/super.c
+++ b/fs/ext4/super.c
@@ -2190,7 +2190,7 @@ static void ext4_orphan_cleanup(struct super_block *sb,
  if (EXT4_SB(sb)->s_mount_state & EXT4_ERROR_FS) {
   /* don't clear list on RO mount w/ errors */
   if (es->s_last_orphan && !(s_flags & MS_RDONLY)) {
-   jbd_debug(1, "Errors on filesystem, "
+   ext4_msg(sb, KERN_INFO, "Errors on filesystem, "
       "clearing orphan list.\n");
    es->s_last_orphan = 0;
   }
-- 
1.7.1